dandiapi/urls.py

```diff
@@ -105,6 +106,7 @@ def to_url(self, value):
     path('admin/', admin.site.urls),
     path('dashboard/', DashboardView.as_view(), name='dashboard-index'),
     path('dashboard/user/<str:username>/', user_approval_view, name='user-approval'),
+    path('dashboard/mailchimp', mailchimp_csv_view, name='mailchimp-csv'),
```

Comment:
```suggestion
    path('dashboard/mailchimp/', mailchimp_csv_view, name='mailchimp-csv'),
```

Django will automatically redirect, but just to be consistent with the rest of the routes.